crates/storage/src/tables/beacon_block.rs

```diff
@@ -34,6 +34,20 @@ impl Table for BeaconBlockTable {
         Ok(result.map(|res| res.value()))
     }
 
+    fn get_all(&self) -> Result<Vec<Self::Value>, StoreError> {
+        let read_txn = self.db.begin_read()?;
+
+        let table = read_txn.open_table(BEACON_BLOCK_TABLE)?;
+        let mut blocks = Vec::new();
+        
+        for entry in table.iter()? {
+            let (_, value) = entry?;
+            blocks.push(value.value());
+        }
+        
+        Ok(blocks)
+    }
```

Comment:
we don't want to read 10 million blocks from disk, the spec says `Retrieves all possible chain heads (leaves of fork choice tree).`